### PR TITLE
added team manifests; other minor edits

### PR DESCRIPTION
**Pull request by youngjeong46**
*Issue #, if available:*

*Description of changes:*

- Added team manifests into teams directory and made changes to the blueprints to leverage them.
- Minor edits include upgrading npm module version under `package.json`.
- Addresses Issue #7 
- README edits to address incorrect links.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/multi-region-construct/index.ts [hunk 1]
```diff
@@ -37,15 +39,19 @@ export default class MultiRegionConstruct {
         
         const blueprint = ssp.EksBlueprint.builder()
             .account(process.env.CDK_DEFAULT_ACCOUNT!)
-            .addOns(new ssp.NginxAddOn,
+            .addOns( new ssp.AppMeshAddOn,
```
Comment (shapirov103) [inline lib/multi-region-construct/index.ts:42]: Do we need appmesh here? Seems exclusive with nginx.

File: lib/multi-team-construct/index.ts [hunk 2]
```diff
@@ -19,21 +21,22 @@ export default class MultiTeamConstruct extends cdk.Construct {
         const teams: Array<ssp.Team> = [
             platformTeam,
             new team.TeamTroiSetup,
-            new team.TeamRikerSetup,
-            new team.TeamBurnhamSetup(scope)
+            new team.TeamRikerSetup(scope, teamManifestDirList[1]),
+            new team.TeamBurnhamSetup(scope, teamManifestDirList[0])
```
Comment (shapirov103) [inline lib/multi-team-construct/index.ts:25]: 👍

File: lib/teams/team-burnham/index.ts [hunk 3]
```diff
@@ -12,13 +12,42 @@ function getUserArns(scope: Construct, key: string): ArnPrincipal[] {
...
-    constructor(scope: Construct) {
+    constructor(scope: Construct, teamManifestDir: string) {
         super({
             name: "burnham",
             users: getUserArns(scope, "team-burnham.users"),
             namespaceAnnotations: {
                 "appmesh.k8s.aws/sidecarInjectorWebhook": "enabled"
-            }
+            },
+            teamSecrets: [
```
Comment (shapirov103) [inline lib/teams/team-burnham/index.ts:22]: Why do we need the auth pwd here?

File: lib/teams/team-burnham/index.ts [hunk 4]
```diff
@@ -12,13 +12,42 @@ function getUserArns(scope: Construct, key: string): ArnPrincipal[] {
...
+                        ]
+                    }
+                }
+            ],
+            teamManifestDir: teamManifestDir
+        });
+    }
+}
+
+class GenerateSecretManagerProvider implements SecretProvider {
```
Comment (shapirov103) [inline lib/teams/team-burnham/index.ts:40]: If this is something that we need across the board, we should move it to quickstart.

**Comment by shapirov103** [comment]
@youngjeong46  were you able to validate any of the modified blueprints by provisioning them?

**Comment by youngjeong46** [comment]
> @youngjeong46 were you able to validate any of the modified blueprints by provisioning them?

All blueprints were validated with the exception of nginx. Will validate with a custom domain.

File: lib/fargate-construct/index.ts [hunk 5]
```diff
@@ -17,7 +17,13 @@ export default class FargateConstruct extends cdk.Construct {
 
             // AddOns for the cluster.
             const addOns: Array<ssp.ClusterAddOn> = [
-                new ssp.ArgoCDAddOn
+                new ssp.AppMeshAddOn,
+                new ssp.AwsLoadBalancerControllerAddOn,
+                new ssp.NginxAddOn,
+                new ssp.ArgoCDAddOn,
+                new ssp.CalicoAddOn,
```
Comment (shapirov103) [inline lib/fargate-construct/index.ts:24]: Calico won't work on Fargate as well as any other CNI solution other than vpc-cni.

File: lib/fargate-construct/index.ts [hunk 6]
```diff
@@ -17,7 +17,13 @@ export default class FargateConstruct extends cdk.Construct {
...
+                new ssp.MetricsServerAddOn,
+                new ssp.SecretsStoreAddOn
```
Comment (shapirov103) [inline lib/fargate-construct/index.ts:26]: Secret store creates a daemonset, so won't work on Fargate as it stands (unless there is an option to deploy is as a regular k8s deployment).

**Review by shapirov103 — COMMENTED** [review]
A few  comments.

File: lib/teams/team-burnham/index.ts (hunk 4 above)
Comment (youngjeong46) [reply lib/teams/team-burnham/index.ts:40]: sounds good. Will do.

File: lib/teams/team-burnham/index.ts (hunk 3 above)
Comment (youngjeong46) [reply lib/teams/team-burnham/index.ts:22]: This is to show how to use secrets store add on. Not necessary, but we can include a sample deployment manifest file to show using the secret as mounted volume.

File: lib/fargate-construct/index.ts (hunk 5 above)
Comment (youngjeong46) [reply lib/fargate-construct/index.ts:24]: addressed.

File: lib/fargate-construct/index.ts (hunk 6 above)
Comment (youngjeong46) [reply lib/fargate-construct/index.ts:26]: addressed.

File: lib/teams/team-burnham/index.ts (hunk 3 above)
Comment (shapirov103) [reply lib/teams/team-burnham/index.ts:21]: sounds good

File: lib/custom-cluster-construct/index.ts [hunk 7]
```diff
@@ -20,13 +19,16 @@ export default class CustomClusterConstruct extends cdk.Construct {
...
-            
+            new ssp.AppMeshAddOn,
+            new ssp.AwsLoadBalancerControllerAddOn,
             new ssp.NginxAddOn,
             new ssp.ArgoCDAddOn,
             new ssp.CalicoAddOn,
             new ssp.MetricsServerAddOn,
+            new ssp.ClusterAutoScalerAddOn,
             new ssp.ContainerInsightsAddOn,
+            new ssp.XrayAddOn,
```
Comment (shapirov103) [inline lib/custom-cluster-construct/index.ts:30]: why do we need xray addon here? It is somewhat exclusive with the AppMeshAddOn which has its own xray support.

**Review by shapirov103 — COMMENTED** [review]
looks good. I was not clear about rationale to add more add-ons to some of the blueprints. What was the reason?

**Review by shapirov103 — APPROVED** [review]
merging , no outstanding issues

**Pull request merged**